tests/callbacks/log_plays.py

```diff
@@ -121,7 +121,9 @@ def __init__(self, name):
             self.isalwaysok = teststat
 
     def update(self, status):
-        self.status = status
+        # We cannot go back to OK after CHANGED
+        if (self.status != 'CHANGED') or (status != 'OK' and self.status == 'CHANGED'):
```

Comment:
Isn't it equivalent to the following?
```python
if self.status != 'CHANGED' or status != 'OK':
```
An `if not` statement might be more explicit here:
```python
if not (self.status == 'CHANGED' and status == 'OK'):
```
@awailly What do you think?